docs/docs/index.md

```diff
@@ -1,34 +1,11 @@
 ---
-title: Building and Running Mesos-DNS
+title: Installing and running Mesos-DNS
 ---
 
-## Building And Running Mesos-DNS
+## Installing and running Mesos-DNS
 
-### Building Mesos-DNS
-
-To build Mesos-DNS, you need to install [`go`](https://golang.org/doc/install) and [`godep`](https://github.com/tools/godep) on your computer. You **must** set the `GOPATH` environment variable to point to the directory where outside `go` packages will be installed. You **must** also add `$GOPATH/bin` to the `PATH` environment variable. If you install go to a custom location, you may need to set the `GOROOT` environment properly and add `$GOROOT/bin` to the `PATH` environment variable. For instance, you may need to do the following:
-
-```
-export GOPATH=$HOME/go
-export PATH=$PATH:$GOPATH/bin
-export GOROOT=/usr/local/go      # assuming go is installed at /usr/local/go
-export PATH=$PATH:$GOROOT/bin
-```   
-
-To build Mesos-DNS using `godep`: 
-
-```
-go get github.com/mesosphere/mesos-dns
-cd $GOPATH/src/github.com/mesosphere/mesos-dns
-make all
-``` 
-
-This generates `mesos-dns`, a statically-linked binary that can be installed anywhere. You will find a sample configuration file `config.json` in the same directory. 
-
-We have built and tested Mesos-DNS with `go` versions 1.3.3 and 1.4. Newer versions of `go` should work as well. 
-
-
-### Running Mesos-DNS
+The official distribution and installation channel is pre-compiled
```

Comment:
Something along the lines of:
"""
Stable binaries for the project are published using via the GitHub release
channel: https://github.com/mesosphere/mesos-dns/releases.
"""

On Mon, Sep 21, 2015 at 1:25 PM, Tomás Senart <notifications@github.com>
wrote:

> In docs/docs/index.md
> <https://github.com/mesosphere/mesos-dns/pull/273#discussion_r39998731>:
>
> > -
> > -To build Mesos-DNS using `godep`:
> > -
> > -```
> > -go get github.com/mesosphere/mesos-dns
> > -cd $GOPATH/src/github.com/mesosphere/mesos-dns
> > -make all
> > -```
> > -
> > -This generates `mesos-dns`, a statically-linked binary that can be installed anywhere. You will find a sample configuration file `config.json` in the same directory.
> > -
> > -We have built and tested Mesos-DNS with `go` versions 1.3.3 and 1.4. Newer versions of `go` should work as well.
> > -
> > -
> > -### Running Mesos-DNS
> > +The official distribution and installation channel is pre-compiled
>
> You're right. Even if installation is just a mv to a known directory,
> this needs rephrasing. What do you suggest?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/mesos-dns/pull/273/files#r39998731>.
>